scripts/nb-tester/test-notebook.py

```diff
@@ -168,6 +189,9 @@ def _execute_notebook(filepath: Path, args: argparse.Namespace) -> nbformat.Note
     if not args.write:
         return nb
```

Comment:
Should this happen after remove_mocking_cell()?

Reply:
No because we don't care about the final state of the notebook if we're not writing the result to disk